src/utils/config.ts

```diff
@@ -93,26 +93,40 @@ export const SpidParams = t.union([
     ContactPersonParams
   ])
 ]);
-
 export type SpidParams = t.TypeOf<typeof SpidParams>;
 
-const JWTParams = t.union([
+export const UserRegistryParams = t.union([
   t.interface({
-    ENABLE_JWT: t.literal(true),
-    JWT_TOKEN_ISSUER: NonEmptyString,
-    JWT_TOKEN_PRIVATE_KEY: NonEmptyString
+    ENABLE_USER_REGISTRY: t.literal(true),
```

Comment:
Since `ENABLE_USER_REGISTRY` is a boolean you should parse it from string env in `IConfig.decode`. Something like 
```
ENABLE_USER_REGISTRY: fromNullable(process.env.ENABLE_USER_REGISTRY)
    .map(_ => _.toLowerCase() === "true")
    .getOrElseL(() => false)
``` 
should suffice.

Reply:
Fix with fdc0da6